packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -591,7 +595,7 @@ export type CallAdapterLocator =
  * @public
  */
 export type AzureCommunicationCallAdapterArgs = {
-  userId: CommunicationUserIdentifier;
+  userId: CommunicationUserIdentifier | /* @conditional-compile-remove(teams-identity) */ MicrosoftTeamsUserIdentifier;
```

Comment:
can you add this to the callwithchat adapter args also?

Reply:
Didn't add it there intentionally.

We can't use CTE token for access ACS chat service, so it doesn't make sense to support this in `CallWithChatComposite` at this point.